frontend/pages/_app.tsx

```diff
@@ -11,17 +13,30 @@ const MyApp = ({ Component, pageProps }) => {
         askNotificationPermissions();
     }, []);
     return (
-        <SWRConfig
-            value={{
-                fetcher: (path, ...args) =>
-                    doApiRequest(path, ...args).then((res) => res.json()),
-                refreshWhenHidden: true,
-            }}
-        >
-            {/* This is necessary for generic HOC */}
-            {/* eslint-disable-next-line */}
-            <Component {...pageProps} />
-        </SWRConfig>
+        <>
+            <SWRConfig
+                value={{
+                    fetcher: (path, ...args) =>
+                        doApiRequest(path, ...args).then((res) => res.json()),
+                    refreshWhenHidden: true,
+                }}
+            >
+                {/* This is necessary for generic HOC */}
+                {/* eslint-disable-next-line */}
```

Comment:
Can we do specific eslint disabling for the next line (for a specific warning) so there's more clarity?